lib/Common/Memory/HeapBucket.h

```diff
@@ -284,6 +289,7 @@ HeapBucketT<TBlockType>::SweepBucket(RecyclerSweep& recyclerSweep, Fn sweepFn)
         // Every thing is swept immediately in non partial collect, so we can allocate
         // from the heap block list now
         StartAllocationAfterSweep();
+        this->newAssignedPageCountDuringBGSweeping = 0;
```

Comment:
changed to use uncollectedNewPageCount so should not zero out here any more. I thought about only count the new pages during sweeping.